src/late/ipfs-on-path/scripts/install.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs-extra')
+const which = require('which')
+const { backup } = require('../../../utils/scripts/backup')
+const getArg = require('../../../utils/scripts/args')
+const { SOURCE_SCRIPT, DEST_SCRIPT } = require('./consts')
+
+// TODO: be careful of file permissions when fs.something()
```

Comment:
@lidel when using `sudo`, we need to be careful with the permissions when using the `fs.*` functions. Could you help me with that, please?

Reply:
Should we `chown` to the user and group running IPFS Desktop process with which set of permissions?

Reply:
`ipfs-on-path/scripts/install.js` will be executed as `root` via `await sudo.exec` in `ipfs-on-path/index.js`, which means it will have all permissions to create the symlink.

AFAIK a symlink itself does not have its own permissions (at least on Linux), instead it inherits permissions of original binary, so if user was able to execute `/some/path/to/ipfs` before, it will also work via `/usr/local/bin/ipfs`

In other words, I don't think we need to do anything additional here (but we need to test if symlinking actually works on a distribution with polkit)